### PR TITLE
[Feature] [transform] To quickly locate the wrong SQL statement in flink sql transform

### PR DESCRIPTION
**Pull request by bigdataf**
…ink sql transform

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-sql/src/main/java/org/apache/seatunnel/flink/transform/Sql.java [hunk 1]
```diff
@@ -42,16 +42,26 @@
...
-    public DataStream<Row> processStream(FlinkEnvironment env, DataStream<Row> dataStream) {
+    public DataStream<Row> processStream(FlinkEnvironment env, DataStream<Row> dataStream) throws Exception {
         StreamTableEnvironment tableEnvironment = env.getStreamTableEnvironment();
-        Table table = tableEnvironment.sqlQuery(sql);
+        Table table = null;
+        try {
+            table = tableEnvironment.sqlQuery(sql);
+        } catch (Exception e) {
+            throw new Exception(sql, e.getCause());
+        }
```
Comment (ruanwenjun) [inline seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-sql/src/main/java/org/apache/seatunnel/flink/transform/Sql.java]: In my knowledge, this PR is hope to catch the exception caused by sql, and add the wrong sql in exception message. If so, suggest adding whole error message.
e.g. `"Flink SQL transform execute failed, SQL " + sql`.
And if we only use `e.getCause()` to transport the exception, we may lose the exception message.

**Review by ruanwenjun — CHANGES_REQUESTED** [review]

File: seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-sql/src/main/java/org/apache/seatunnel/flink/transform/Sql.java (hunk 1 above)
Comment (bigdataf) [reply seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-sql/src/main/java/org/apache/seatunnel/flink/transform/Sql.java]: @ruanwenjun 
Yes,thanks for your advice，you mean for example like this?
`throw new Exception("Flink SQL transform execute failed, SQL："+sql, e);`

Comment (ruanwenjun) [reply seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-sql/src/main/java/org/apache/seatunnel/flink/transform/Sql.java]: Yes, or you can change the error message.

Comment (bigdataf) [reply seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-sql/src/main/java/org/apache/seatunnel/flink/transform/Sql.java]: thank you @ruanwenjun, i'll change it

Comment (bigdataf) [reply seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-sql/src/main/java/org/apache/seatunnel/flink/transform/Sql.java]: Please review，Thank you

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Review by wuchunfu — APPROVED** [review]
LGTM

**Pull request merged**